packages/components/src/Socket.js

```diff
@@ -0,0 +1,96 @@
+import { useEffect } from 'react'
+
+const STORAGE_INDEX = 'mdx-slide'
+const STORAGE_STEP = 'mdx-step'
```

Comment:
If these aren't in a shared `constants` module, these probably should be if the `localStorage` logic is still anywhere else

Reply:
I will move these to the `constants` module